wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -1,117 +1,82 @@
+						<p><a class="btn btn-primary" href="https://wordpress.org/plugins/engaging-buttons/">Download Plugin</a> <a class="btn btn-default" href="https://github.com/engagingnewsproject/engaging-button/">View Source</a></p>
 					</div>
-				</section>
-
-				<section class="quiz-creator-benefits">
-					<div class="container container--xl wrapper--pad">
-
-						<!-- related research widget -->
-						<h3 class="text-center">Everything you expect in a quiz creator and much more.</h3>
-						<div class="tile-stats">
-							<div class="tile">
-								<h4>Easy to use</h4>
-								<i class="fas fa-universal-access"></i>
-								<p>Step-by-step process makes it easy for anyone to create a quiz. Create a quiz in as little as 3 minutes.</p>
-							</div>
-							<div class="tile">
-								<h4>Customizable</h4>
-								<i class="fas fa-palette"></i>
-								<p>Customize colors to match your branding and seamlessly add it to your site.</p>
-							</div>
-							<div class="tile">
-								<h4>Embeddable</h4>
-								<i class="fas fa-code"></i>
-								<p>Easily add the quiz to any page on your website with our simple embed code.</p>
-							</div>
-							<div class="tile">
-								<h4>Gain insight</h4>
-								<i class="fas fa-chart-pie"></i>
-								<p>Test your audience’s knowledge and response rate. Even create A/B tests to see which quizzes are more effective.</p>
-							</div>
-						</div>
+					<div class="col-md-6">
+					<figure><img class="aligncenter size-full wp-image-1340" src="/wp-content/uploads/2016/01/Engaging-Buttons-Plugin.jpg" alt="Engaging Buttons Plugin Example" width="460" height=""></figure>
 					</div>
-				</section>
+				</div>
+			</section>
+			<section>
+				<div class="row">
+					<div class="col-md-6">
+						<div class="section-header">
+							<h2>Quiz Creator</h2>
+							<h4>Create a quiz to engage your users. Easily embed the quiz on any website.</h4>
+						</div>
+						<p><!--
+
 
-				<section class="callout quiz-creator-research">
-					<div class="container container--xl wrapper--pad">
 
-							<h4 class="subhead">A quiz tool designed to increase civic engagement</h4>
-							<h2 class="headline">Challenge your Audience</h2>
-							Our research shows that quizzes help people learn more than presenting information without a quiz. Even more, they increase time on page and site visitors rate them as enjoyable.
+				<figure><img class="aligncenter size-full wp-image-1334" src="https://mediaengagement.org/wp-content/uploads/2016/01/quiz-tool-example.png" alt="Quiz tool example question" width="460" /></figure>
 
-							<a href="/research/online-polls-and-quizzes/">Explore White Paper</a>
+				 --></p>
+						<p>Our <a href="http://mediaengagement.org/research/online-polls-and-quizzes/">research on quizzes</a> shows that quizzes help your users:</p>
```

Comment:
Since we're processing this in twig, Might as well replace the hardcoded links with something like `{{ site.url }}` so it works correctly on all our environments.